Cython/Compiler/Pipeline.py

```diff
@@ -188,8 +188,9 @@ def create_pipeline(context, mode, exclude_classes=()):
         ParallelRangeTransform(context),
         AdjustDefByDirectives(context),
         WithTransform(context),
```

Comment:
I think we should also try to swap these two. The `AdjustDefByDirectives` is highly related to those that follow now.
```suggestion
        WithTransform(context),
        AdjustDefByDirectives(context),
```